eventstore/tasks.py

```diff
@@ -319,6 +323,76 @@ def update_rapidpro_contact_error_timestamp(context):
     )
 
 
+@app.task(
+    autoretry_for=(RequestException, ConnectionException),
+    retry_backoff=True,
+    max_retries=15,
+    acks_late=True,
+    soft_time_limit=10,
+    time_limit=15,
+)
+def upload_and_delete_cdu_address_updates(self):
+
+    Headers = [
+        "msisdn",
+        "first_name",
+        "last_name",
+        "id_type",
+        "id_number",
+        "date_of_birth",
+        "folder_number",
+        "district",
+        "municipality",
+        "city",
+        "suburb",
+        "street_name",
+        "street_number",
+    ]
+    data = []
+    ids_to_delete = []
+    for address_update in CDUAddressUpdate.objects.all():
+        ids_to_delete.append(address_update.id)
+        row = ",".join([str(getattr(address_update, header)) for header in Headers])
+        data.append(row)
+
+    # Write to csv
+    with open("./CDU_address.csv", "w", newline="") as csv_file:
```

Comment:
There is a `tempfile` library in python we can use to create a temporary file to write to, instead of relying on a static filename, where we might not have write access to.